ods_ci/tests/Resources/Files/accelerator/qe_create_ap_2.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: dashboard.opendatahub.io/v1
+kind: AcceleratorProfile
+metadata:
+  name: qecreateap2
+  namespace: redhat-ods-applications
+spec:
+  description: description example
+  displayName: qe_create_ap_2
```

Comment:
could you try with a more realistic displayed name? e.g., QE Accelerator Profile or QE GPU Accelerator etc

Reply:
When a user creates an Accelerator Profile using the UI, when it's asked about the name of that AP, actually the value of that field is the displayName and the name is transformed  eliminating the hyphens and blank spaces. In addition I wanted to maintain certain relation between the name, the displayname and the name of the yaml file used (for that reason I don't use spaces either, so that when doing an "oc create -f xxxx.yaml" it is not necessary to escape the spaces). Besides, keeping this relation between name, displayname and file name, makes it easier to see which resources are used in each test.
Still I agree with you that it is not a good name, so I have changed it to _qeacceleratorprofileX_ and _qe_accelerator_profile_X_.
Fixed in  a18437b